### PR TITLE
added a placeholder texture, and moved us to a versioned folder layout

### PR DESCRIPTION
**Pull request by xyzzy121**

### CODE REVIEW

File: 1.5/Mods/Biotech/Defs/aotcGeneProcessors.xml [hunk 1]
```diff
@@ -0,0 +1,211 @@
+<Defs>
+  <ThingDef ParentName="BuildingBase">
```
Comment (Para0234) [inline 1.5/Mods/Biotech/Defs/aotcGeneProcessors.xml:3]: When creating something requiring a DLC, don't forget the tag "MayRequire="[Name of the mod]""

Here, I think the line should look like   \<ThingDef ParentName="BuildingBase"  MayRequire="Ludeon.Rimworld.Biotech">

File: 1.5/Mods/Biotech/Defs/aotcGeneProcessors.xml [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+      </li>
+      <li Class="CompProperties_Facility">
+        <maxSimultaneous>100</maxSimultaneous>
+        <maxDistance>18</maxDistance>
+        <showMaxSimultaneous>false</showMaxSimultaneous>
+      </li>
+    </comps>
+  </ThingDef>
+  
+  <ThingDef ParentName="BuildingBase">
```
Comment (Para0234) [inline 1.5/Mods/Biotech/Defs/aotcGeneProcessors.xml:143]: Same as above, we don't want the building accessible without biotech enabled.

Here, I think the line should look like \<ThingDef ParentName="BuildingBase"  MayRequire="Ludeon.Rimworld.Biotech">

**Review by Para0234 — CHANGES_REQUESTED** [review]
Only issue is that we didn't think of the MayRequire="Ludeon.Rimworld.Biotech" to prevent the buildings from being accessible without the DLC.

**Comment by Para0234** [comment]
The changes requested should be redundant with another change of the PR. I'll verify it as soon as I have the time to do so.

**Comment by xyzzy121** [comment]
alright, so they are implemented, (although using a placeholder texture.) the only thing left now is to balance them.

**Review by Para0234 — APPROVED** [review]

**Pull request merged**